### ISSUE TITLE
Support top-level functionality mode option

### ISSUE THREAD

**Issue by rarkins** [renovatebot/renovate#15865]
### What would you like Renovate to be able to do?

Multiple people have now requested "vulnerability fixes only", and while the config can be done, it is a little complex: https://github.com/renovatebot/renovate/discussions/15490#discussioncomment-2702421

I would like to make this a simpler, first-class option.

### If you have any ideas on how this should be implemented, please tell us here.

To keep things really simple, I would like to propose combining this with dry run, e.g. a new option:

```
  "mode": "silent" | "remediation" | "interactive"
```

mode=silent would map to dryRun=lookup (i.e. extract dependencies, identify updates, but don't apply any)

mode=remediation means only fix vulnerable issues (rest could be left for dependency dashboard perhaps)

mode=interactive is default, normal behavior.

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by nabeelsaabna** [comment]
since `dryRun` defined for self-hosted, will the new `mode` option be available for all ?

**Comment by rarkins** [comment]
Yeah there's some overlap there. Mode would be like a higher level concept than dry run. 

For example it should be possible to do remediation mode with full dry run still.

Although there's a little overlap or chance of confusion, I still like this concept
